django/contrib/admin/helpers.py

```diff
@@ -304,6 +308,14 @@ def inline_formset_data(self):
         })
 
     @property
+    def forms(self):
+        return self.formset.forms
+
+    @property
+    def non_form_errors(self):
+        return self.formset.non_form_errors
```

Comment:
You're right, the diff doesn't show up the difference, but If that method is for `InlineAdminFormSet` what about non_form_errors with an AdminForm?